app/src/main/res/values-night/themes.xml

```diff
@@ -1,6 +1,6 @@
 <resources xmlns:tools="http://schemas.android.com/tools">
     <!-- Base application theme. -->
-    <style name="Theme.Sharingang" parent="Theme.MaterialComponents.DayNight.DarkActionBar">
+    <style name="Theme.Sharingang" parent="Theme.MaterialComponents.DayNight.NoActionBar">
```

Comment:
What does this do?

Reply:
It removes the action bar that used to say Sharingang at the top of the app